aopy/visualization/animation.py

```diff
@@ -207,3 +215,89 @@ def plotdata(i):
     ani = FuncAnimation(fig, plotdata, 
                         frames=len(cursor_trajectory), interval=1000./samplerate)  
     return ani
+
+def get_animate_circles_func(samplerate, bounds, circle_radii, circle_colors, *circle_ts, history=1., ax=None):
+    '''
+    Draws an animation of an arbitrary number of circles. Used in :func:`~aopy.visualization.animation.animate_behavior`.
+
+    Example:
+
+        .. raw:: html
+
+            <video controls src="_static/demo_anim.mp4"></video>
+
+    Args:
+        samplerate (float): The sampling rate of the trajectories in Hz.
+        bounds (tuple): Boundaries of the plot area. See :func:`~aopy.visualization.plot_targets`.
+        circle_radii (float): Radius of the targets.
+        circle_color (plt.color, optional): Color of the eye trajectory. Default is 'purple'.
+        circle_ts ((nt, ndim) array): Cursor positions over time for 2D or 3D trajectories.
+        history (float, optional): time in seconds to maintain the trajectory. Default 0.
```

Comment:
Default looks like it is 1. I also don't understand what this does by the description.